### DISCUSSION TITLE
How configure nitro to support hybrid rendering?

### DISCUSSION THREAD

**Discussion by SergioEanX** [nuxt/nuxt#16521] [Questions]
I was not able to verify hybrid rendering on my local machine.
At Nuxt official [documentation](https://disq.us/url?url=https%3A%2F%2Fv3.nuxtjs.org%2Fguide%2Fconcepts%2Frendering%23hybrid-rendering%3Ai7GXaGm2H5xAWDXPv2q2_6Nyzio&cuid=4942816)) is stated that:

> Whenever possible, route rules will be automatically applied to the
> deployment platform's native rules (currently **Netlify and Vercel are supported**).

I tried to build a Nuxt 3 app an run it using `node .output/server/index.mjs`, but this configuration does not support hybrid rendering. What additional steps must be performed to enable/support hybrid rendering using nitro?

**Comment by TheAlexLichter** [comment]
As quoted correctly from the docs (and also from https://github.com/nuxt/framework/discussions/8296), hybrid rendering is only available via Netlify and Vercel right now. Further support is WIP.
